pkg/libcontainer/libcontainer_linux.go

```diff
@@ -0,0 +1,104 @@
+//go:build linux
+// +build linux
+
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package libcontainer
+
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import (
+	libcontainercgroups "github.com/opencontainers/runc/libcontainer/cgroups"
+	libcontainerfscommon "github.com/opencontainers/runc/libcontainer/cgroups/fscommon"
+	libcontainermanager "github.com/opencontainers/runc/libcontainer/cgroups/manager"
+	libcontainersystemd "github.com/opencontainers/runc/libcontainer/cgroups/systemd"
+	libcontainerconfigs "github.com/opencontainers/runc/libcontainer/configs"
+	libcontainerutils "github.com/opencontainers/runc/libcontainer/utils"
+)
+
+var (
+	IsCgroup2UnifiedMode        = libcontainercgroups.IsCgroup2UnifiedMode
+	IsNotFound                  = libcontainercgroups.IsNotFound
+	HugePageSizes               = libcontainercgroups.HugePageSizes
+	GetPids                     = libcontainercgroups.GetPids
+	FindCgroupMountpointAndRoot = libcontainercgroups.FindCgroupMountpointAndRoot
+	GetOwnCgroup                = libcontainercgroups.GetOwnCgroup
+	ParseCgroupFile             = libcontainercgroups.ParseCgroupFile
+	NewNotFoundError            = libcontainercgroups.NewNotFoundError
+	GetAllSubsystems            = libcontainercgroups.GetAllSubsystems
+	PathExists                  = libcontainercgroups.PathExists
+
+	New                  = libcontainermanager.New
+	GetCgroupParamUint   = libcontainerfscommon.GetCgroupParamUint
+	GetCgroupParamString = libcontainerfscommon.GetCgroupParamString
+
+	ExpandSlice = libcontainersystemd.ExpandSlice
+
+	CleanPath = libcontainerutils.CleanPath
+)
+
+type Mount libcontainercgroups.Mount
```

Comment:
would type aliases (e.g. `type Mount = libcontainercgroups.Mount`)  avoid the need for casting?